providerframework/metaservice/archiver/manager/manager_test.go

```diff
@@ -0,0 +1,168 @@
+/*
+Copyright 2022 The SODA Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package manager
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/soda-cdm/kahu/providerframework/metaservice/archiver"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+)
+
+type fakeStruct struct{}
```

Comment:
IMO, If ```fakeStruct``` is used in multiple packages, Adding a utility is more meaningful.

Reply:
added mock path in which mocks are added